lib/postgrex/replication.ex

```diff
@@ -836,6 +690,23 @@ defmodule Postgrex.Replication do
           :ok -> {:noreply, s}
           {error, reason, protocol} -> reconnect_or_stop(error, reason, protocol, s)
         end
+
+      {:query, query, mod_state} ->
+        case s do
+          %{repl_opts: nil, copy_opts: nil} ->
+            case Protocol.handle_simple(query, [], s.protocol) do
+              {:ok, %Postgrex.Result{} = result, protocol} ->
+                handle(mod, :handle_result, [result, mod_state], %{s | protocol: protocol}, from)
+
+              {error, reason, protocol} ->
+                from && reply(from, {error, reason})
+                reconnect_or_stop(error, reason, protocol, %{s | state: {mod, mod_state}})
+            end
+
+          _ ->
+            reply(from, {:error, :stream_in_progress})
+            raise "cannot perform queries after replication or copying has started"
```

Comment:
I think just the reply should be good, otherwise it will kill the replication/copying?

Reply:
I think it is fine to raise because it is an error in the application logic most likely? Especially because queries come from within.

Reply:
Makes sense especially with the queries coming from the process.